bears/python/PEP8Bear.py

```diff
@@ -39,6 +37,8 @@ def run(self, filename, file,
         :param local_pep8_config: Set to true if autopep8 should use a config
                                   file as if run normally from this directory.
         """
+        autopep8 = list(self.__class__.REQUIREMENTS)[0]
```

Comment:
dangerous, somebody might add something to the set and out of a sudden it might fail to work with some interpreters or completely nondeterministic